Core/automation/lib/python/core/utils.py

```diff
@@ -33,13 +35,15 @@
 try:
     from org.joda.time import DateTime as JodaDateTime
 except:
+    # OH3 does not have Joda Time
```

Comment:
Don't modify other parts of the code, muddies the water if a PR adding a function does random other things too.

Reply:
Removed